basalt/autograd/ops/basics.mojo

```diff
@@ -718,7 +718,8 @@ struct FMA:
     fn result_shape(
         t1_shape: TensorShape, t2_shape: TensorShape, t3_shape: TensorShape
     ) -> TensorShape:
-        return broadcast_shapes(t1_shape, t2_shape, t3_shape)
+        # FMA assumes: t1_shape == t2_shape == t3_shape
+        return t1_shape
```

Comment:
We should start commenting `# TODO: Error Handling` on places like these for when we eventually add some assertion logic.
```suggestion
        # FMA assumes: t1_shape == t2_shape == t3_shape
        # TODO: Error Handling
        return t1_shape
```

Reply:
Yeah, the API should handle this imo. To keep the kernels clean